messagediff.go

```diff
@@ -158,15 +158,25 @@ func (d *Diff) diff(aVal, bVal reflect.Value, path Path) bool {
 			if field.Tag.Get("testdiff") == "ignore" { // skip fields marked to be ignored
 				continue
 			}
+			skip := false
+			for _, ig := range ignores {
+				if field.Name == ig {
+					skip = true
+					break
```

Comment:
I think it would be a lot cleaner to put a label on the outer loop and call `continue outer` instead  of having this skip variable.